### PR TITLE
Rename "mixer" to "audioengine" on lmmsrc.xml

### PR DESCRIPTION
**Pull request by M374LX**
Although #6127 is now merged, the old name "mixer" persists in the lmmsrc.xml configuration file.

### CODE REVIEW

**Review by JohannesLorenz — APPROVED** [review]

**Comment by JohannesLorenz** [comment]
Thanks for your contribution. Will merge in 3 days if no one screams.

File: src/core/ConfigManager.cpp [hunk 1]
```diff
@@ -40,7 +40,9 @@
 
 // Vector with all the upgrade methods
 const std::vector<ConfigManager::UpgradeMethod> ConfigManager::UPGRADE_METHODS = {
-	&ConfigManager::upgrade_1_1_90    ,    &ConfigManager::upgrade_1_1_91
```
Comment (Spekular) [inline src/core/ConfigManager.cpp:43]: I think these should stay on one line since that's what we do in the project file upgrader.

Comment (Spekular) [reply src/core/ConfigManager.cpp:43]: To be clear, not all on one line, but 2 per line.

File: src/core/ConfigManager.cpp [hunk 2]
```diff
@@ -119,16 +121,46 @@ void ConfigManager::upgrade_1_1_90()
...
+	{
 		setValue("app", "displaydbfs", value("app", "displaydbv"));
 		deleteValue("app", "displaydbv");
 	}
 }
 
+void ConfigManager::upgrade_1_2_2()
+{
+	// Rename mixer to audioengine
+	if (!value("mixer", "audiodev").isNull())
```
Comment (Spekular) [inline src/core/ConfigManager.cpp:137]: What about using a loop or lambda here?

**Pull request merged**